app/controllers/omniauth_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OmniauthController < ApplicationController
+
+  skip_before_filter :verify_authenticity_token, only: :callback
+  skip_before_filter :authenticate_user, only: :callback
+  skip_authorization_check :only => :callback
+
+  def callback
+    auth = request.env['omniauth.auth']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.